src/app/api/quality_matrix/quality_matrix.py

```diff
@@ -258,6 +262,21 @@ def missing_fields(
     return {replication_source: missing_fields_ratio(value, total_count)}
 
 
+async def stored_in_timeline(data: QUALITY_MATRIX_RETURN_TYPE, database: Database):
+    if not await has_table("timeline"):
+        await create_timeline_table()
+
+    _, timeline_table = await get_table()
+    insert_statement = timeline_table.insert().values(
+        timestamp=datetime.now().timestamp(), quality_matrix=data
+    )
+
+    await database.connect()
+    result = await database.fetch_all(insert_statement)
+    if result is None:
```

Comment:
result will never be none, but an empty list (at least according to database package documentation), instead check for `len(result) == 0`

Reply:
Good point - changed it